### PR TITLE
Update DOCS.md corresponding to #43

### PR DESCRIPTION
**Pull request by grahovam**
Update DOCS.md corresponding to #43

### CODE REVIEW

File: DOCS.md [hunk 1]
```diff
@@ -48,6 +48,9 @@ publish:
     tag: $$BRANCH
 ```
 
+If you use the `$$TAG` variable, you need to enable builds for tags, which can be done in your drone repository setting screen.
+You can also do `$${TAG##v}` to remove the `v` prefix on your tag name.
```
Comment (tboerger) [inline DOCS.md]: I would prefer a link to the docs for the general string substitution

**Comment by tboerger** [comment]
This documentation also matches for drone-ecr and drone-gcr

File: DOCS.md (hunk 1 above)
Comment (grahovam) [inline DOCS.md]: Where can I find those?

**Comment by bradrydzewski** [comment]
I'm closing this because it is documented globally for all plugins at http://readme.drone.io/usage/variables/#string-interpolation:2b8b8ac4006be88c769f5e3fd99b009a

@tboerger I don't want to put the direct link in the docs either for a few reasons. If we put a directly link here, then we need to put the link in the docs for the other 50 plugins. And then we should probably include a link to the .drone.sec file documentation for all 50 plugins. And then think how annoying it will be when we change one of those URLs and have to update 50+ files :smile:  

If anything, we should put links in the master template so that it should up automatically for all of the plugins.

**Pull request closed without merging**